bdk-ffi/src/wallet.rs

```diff
@@ -65,6 +65,11 @@ impl Wallet {
         self.get_wallet().network()
     }
 
+    /// Return whether or not a script is part of this wallet (either internal or external).
+    pub(crate) fn is_mine(&self, script: Arc<Script>) -> Result<bool, BdkError> {
+        Ok(self.get_wallet().is_mine(&script.script).unwrap())
```

Comment:
Can we return the error here instead of the unwrap()? 

```suggestion
        self.get_wallet().is_mine(&script.script)
```

Reply:
Looks good! One suggestion above.